ci-operator/step-registry/gcp/provision/private-dns-zone/in-host-project/gcp-provision-private-dns-zone-in-host-project-commands.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+echo "Reading variables from ${SHARED_DIR}/xpn.json..."
+NETWORK=$(jq -r '.clusterNetwork' "${SHARED_DIR}/xpn.json")
+CLUSTER_PVTZ_PROJECT=$(jq -r '.hostProject' "${SHARED_DIR}/xpn.json")
+
+CLUSTER_NAME="${NAMESPACE}-${JOB_NAME_HASH}"
+
+if [[ -s "${SHARED_DIR}/xpn.json" ]] && [[ -f "${CLUSTER_PROFILE_DIR}/xpn_creds.json" ]]; then
```

Comment:
The step name indicates this step is only working for xpn case, we do not need the if condition any more.

Reply:
It's copied from other provision/deprovision steps, e.g. `gcp-provision-bastionhost`, for QE's gcp profile, because we don't have the private of DEV's gcp profile.

Reply:
Thanks for the info, that reminds me some more ideas. 
Before when we are starting to develop UPI XPN cases using qe's cluster, because `upi-conf-gcp-xpn` is using hard code, we have to do that the copy for compatibility. Now we are the 1st contributor for IPI XPN, so I think we can make the process to be more better.
We can talk it a bit later on slack.